sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/NestedColumnAliasingSuite.scala

```diff
@@ -261,6 +261,37 @@ class NestedColumnAliasingSuite extends SchemaPruningTest {
     comparePlans(optimized, expected)
   }
 
+  test("Do not generate redundant aliases if parent nested field is aliased too") {
```

Comment:
Shall we add a JIRA prefix in the title?

Reply:
Ok.